auth-svc/service/db-service/UserRepository.js

```diff
@@ -0,0 +1,32 @@
+const { PrismaClient } = require('@prisma/client');
+
+const prisma = new PrismaClient();
+
+class UserRepository {
+  async createUser(registrationPayload) {
+    const user = await prisma.users.create({
+      data: {
+        email: registrationPayload.email,
+        password: registrationPayload.password,
+        name: registrationPayload.name,
+        last_name: registrationPayload.last_name,
+        activationlink: registrationPayload.activationLink,
+        role: registrationPayload.role,
+      },
+    });
+
+    return user;
+  }
+
+  async findByEmail(email) {
+    const user = await prisma.users.findUnique({
+      where: {
+        email: email,
```

Comment:
if the key and param are the same, i think you can just do 
```
where: { email }
```
like that.

Reply:
Done